playground/__tests__/Components.test.tsx

```diff
@@ -1,7 +1,13 @@
-import { fireEvent, render, screen, cleanup, within } from "@testing-library/react";
-import '@testing-library/jest-dom'
+import {
```

Comment:
This is only made prettier using `npx prettier --write .`. I wasn't able to undo that step. I can manually remove those files from the PR if you want.